src/components/TableHead.js

```diff
@@ -98,7 +99,7 @@ function TableHead(props) {
       })}>
       <TableHeadRow>
         <TableSelectCell
-          ref={el => setCellRef(0, findDOMNode(el))}
+          ref={el => setCellRef(0, 0, findDOMNode(el))}
```

Comment:
I think we should try to get rid of A findDOMNode

Reply:
I think I found a way. ( ref callback )
I'll commit it soon.

Reply:
I think that may be the last one too, if you figure out a way just submit a PR and I'll merge it into this branch.